packages/paste-core/components/inline-code/stories/index.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {InlineCode} from '../src';
+import type {Story} from '@storybook/react';
+import {Text} from '@twilio-paste/text';
+import {Stack} from '@twilio-paste/stack';
+import {useTheme} from '@twilio-paste/theme';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Inline Code',
+  component: InlineCode,
+};
+
+export const Default: Story = () => {
+  return <InlineCode>0000001111111122234444444</InlineCode>;
+};
+
+export const ReallyLongContent: Story = () => {
+  return (
+    <InlineCode>
+      000 00 0111111 1122 2344444 44 000 00 0111111 1122 2344444 44 000 00 0111111 1122 2344444 44 000 00 0111111 1122
+      2344444 44
+    </InlineCode>
+  );
+};
+
+export const InlineWithText: Story = () => {
+  return (
+    <Text as="span">
+      If the composite has a single row or column (one-dimensional), the <InlineCode>orientation</InlineCode> value
+      determines which arrow keys can be used to move focus.
+    </Text>
+  );
+};
+
+export const WithTextSizes: Story = () => {
+  return (
+    <Stack orientation="vertical" spacing="space60">
+      <Text fontSize="fontSize50" as="div">
+        If the composite has a single row or column (one-dimensional), the <InlineCode>orientation</InlineCode> value
+        determines which arrow keys can be used to move focus.
+      </Text>
+      <Text fontSize="fontSize10" as="div">
+        If the composite has a single row or column (one-dimensional), the <InlineCode>orientation</InlineCode> value
+        determines which arrow keys can be used to move focus.
+      </Text>
+    </Stack>
+  );
+};
```

Comment:
praise: some good edge case stories for VRT here. If you wanted to expand them a little,  setting a large line hieght would prove your line-height inherit worked too.